docs/features/optin.md

```diff
@@ -0,0 +1,56 @@
+# Opt in to Use Alpha Version Controllers
+
+> `Feature State`: `alpha` as of version v1.125
+
+KCC can be configured to enable alpha version of controllers (referred to as
+"direct" controllers) for reconciling specific resources. This will allow users
+to test the new direct controller code for bug fixes or features not available
+in legacy controllers. It is possible to enable the direct controller code on a
+per-resource basis, by setting an annotation for each custom resource.
+
+
+## Enabling
```

Comment:
Could you also explain what resource can apply this annotation (has both TF/DCL and direct supports) ?

Reply:
Ok, added a section "Applicability" for this